client/landing/stepper/declarative-flow/internals/global.scss

```diff
@@ -368,6 +368,7 @@ button {
 		display: none;
 	}
 	.loading-bar {
+		background-color: #e5cfe8;
```

Comment:
Nit: it would be good to push this into a CSS variable so we can describe what the colour is. I _think_ we should be able to use `var(--studio-woocommerce-purple-5)` directly.

Reply:
Thanks! Indeed. Updated in 60d4befa8bf5c605877e0fb59b75be0e4f9735ed